Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:current
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY package.json .
```

Comment:
Why running `COPY . . ` directly here (instead of doing it later) ?

Reply:
It could. I just used a previous docker file where there where a building docker (where deps and build where done) and a running docker (lighter to just run the producted code)